test/torchaudio_unittest/datasets/utils_test.py

```diff
@@ -1,36 +1,38 @@
+import torch
 from torchaudio_unittest.common_utils import (
     TorchaudioTestCase,
-    get_asset_path,
+    TempDirMixin,
+    get_asset_path
```

Comment:
Could you remove this `get_asset_path`? I think it is no longer used.
Also can you remove the files under `test/torchaudio_unittest/assets/CommonVoice/`?
Finally, we can get rid of it.

Reply:
Done, thanks for the review.